spec/lib/govuk/markdown_renderer_spec.rb

```diff
@@ -0,0 +1,180 @@
+require 'rails_helper'
+
+# Ignore whitespace
+def expect_equal_ignoring_ws(first, second)
+  expect(first.lines.map(&:strip).join('')).to eq(second.lines.map(&:strip).join(''))
+end
+
+describe Govuk::MarkdownRenderer do
+  let(:html) { Govuk::MarkdownRenderer.render(markdown) }
+
+  context 'ul' do
```

Comment:
Could we make these `describe` please? As-in, this is the subject of the test.

Reply:
Done